pennylane/gradients/finite_difference.py

```diff
@@ -167,24 +172,56 @@ def _processing_fn(results, shots, single_shot_batch_fn):
     return tuple(grads_tuple)
 
 
-@gradient_transform
+def _expand_transform_finite_diff(

```

Comment:
The expand fn for finite diff, param shift, param shift cv, and spsa grad is the same. It might be worth putting it all in `gradient_transforms.py` and importing it with the rest of the imports

Reply:
In fact that is not possible because we force the expand function to have the same signature as the transform itself.